lib/include/public/Enums.hpp

```diff
@@ -129,6 +129,10 @@ namespace MAT_NS_BEGIN
         PiiKind_Fqdn = 12,
         /// <summary>A legacy IPV4 Internet address.</summary>
         PiiKind_IPv4AddressLegacy = 13,
+        /// <summary>A legacy IPV6 Internet address.</summary>
+        IPv6ScrubLastHextets = 14,
+        /// <summary>Drop the value.</summary>
+        DropValue = 15,
```

Comment:
should these both be defined with `PiiKind` prefix to maintain consistency?

Reply:
Yes. Fixed.